content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,100 @@
 title: "Official Providers"
 weight: 20
 ---
-Upbound creates, maintains and supports a set of Crossplane providers called *official providers*.  
+Upbound creates, maintains and supports a set of Crossplane providers called **official providers**.  
 
-## Find official providers
-Identify official providers in the marketplace with the `by Upbound` gold seal.
+Upbound builds, tests and supports official providers. The Upbound official providers are open source under the Apache 2.0 license. Upbound recommends the official providers for all deployments.
 
-{{<img src="upbound-marketplace/images/provider-by-upbound.png" alt="example official provider with a by Upbound seal" size="xtiny">}}
+Find official providers in the [Upbound Marketplace](https://marketplace.upbound.io/providers?tier=official). 
+All official providers have a purple `Official` tag.
```

Comment:
```suggestion
All official providers have an `Official` tag to identify them.
```

Let's not mention the color so we don't have to update the docs if there's a design change at some stage.